wiz-outpost-lite/templates/deployment.yaml

```diff
@@ -15,10 +15,12 @@ spec:
   template:
     metadata:
       annotations:
-      {{- with .Values.podAnnotations }}
+        {{- with .Values.podAnnotations }}
         {{- toYaml . | nindent 8 }}
-      {{- end }}
-        container.apparmor.security.beta.kubernetes.io/{{ printf "%s-%s" .Chart.Name .runner }}: unconfined
+        {{- end }}
+        {{- with .Values.podAnnotationsApparmor }}
```

Comment:
Inline this, it should be as simple as `{{ if .Values.apparmorProfile }}[...]{{ end }}`